### ISSUE TITLE
Colorcoded items background causing highlighting issues

### ISSUE THREAD

**Issue by Sawtaytoes** [vprimachenko/Sublime-Colorcoder#32]
When doing an outline on text like if I'm in a find operation, the outline does not go around the colorcoded items because they have a background color that matches the unhighlighted background color of the syntax method.

Is it possible to make it so colorcoded items don't have this background color?

**Comment by vprimachenko** [comment]
no. current api does not allow it or to be precise setting the background color is the only way to color only the text<sup>*</sup>. without this "hack" colorcoder would look like this:

![colorcoder-no-bg](https://cloud.githubusercontent.com/assets/1255913/4835435/30ad797e-5fb7-11e4-8642-fb15897b2d86.png)

i am actually frustrated by this as well, as this limitation also interferes with current line-highlighting. i even tried to work around this issues yet the only way i could think of introduced massive lag to the editor so i never pushed it to master.

so as much as i would love to implement this now, i cant… i check the api of every release and will implement this as soon as the api allows.

im not sure if i should close this issue as an "cant-fix" or keep it open 'till the fix arrives.

<sup>*</sup> for whatever reason if an region class does not have an background setting, its foreground is used to fill the background, and the text keeps it color. interestingly enough sublime also does this if background matches the overall theme background. i am puzzled why sublime works this way.

**Comment by Sawtaytoes** [comment]
Darn, that's a good point. Seems like an awfully strange limitation; then again, ST3 has some other long-standing bugs at this time too :/. Sad.

**Comment by Sawtaytoes** [comment]
Is it possible to manually fix this in the modified (Colorcoded).tmTheme files?

**Comment by vprimachenko** [comment]
dont think so, i tried a lot of promising combinations of `findHighlight` and `findHighlightForeground` values yet nothing satisfactory
